core/station/impl/src/models/resource.rs

```diff
@@ -376,24 +371,46 @@ impl Resource {
                         ExternalCanisterResourceAction::Create,
                     )]
                 }
-                ExternalCanisterResourceAction::Change(
-                    ChangeExternalCanisterResourceTarget::Any,
-                ) => {
+                ExternalCanisterResourceAction::Fund(ExternalCanisterId::Any) => {
+                    vec![
+                        Resource::ExternalCanister(ExternalCanisterResourceAction::Fund(
+                            ExternalCanisterId::Any,
+                        )),
+                        // The following additional resources also enable the user to perform the `Fund` action.
```

Comment:
Why?

Reply:
Should a user that can request for changes to the canister not be able to request for it to be funded?

Reply:
Now I understand the motivation - makes sense to me!